libs/ui/src/lib/page-banner/page-banner.tsx

```diff
@@ -0,0 +1,27 @@
+import cn from './page-banner.module.css';
+import EllipseSVG from './svg/ellipse-1.svg';
+import RectangleSVG from './svg/rectangle-1.svg';
+
+interface PageBannerProps {
+  title: string;
+  subtitle: string;
+}
+
+const PageBanner = ({title, subtitle}: PageBannerProps) => {
+  return (
+    <div className={cn['banner']}>
+      <div className={cn['svg-ellipse']}>
+        <EllipseSVG />
+      </div>
+      <div className={cn['title-container']}>
```

Comment:
Det anbefales å bruke camelCase og object style. For eksempel cn.titleContainer og cn.banner.